vendor/modules.txt

```diff
@@ -72,13 +72,14 @@ github.com/hashicorp/hcl/json/scanner
 github.com/hashicorp/hcl/json/token
 # github.com/hashicorp/vault/api v1.0.5-0.20200519221902-385fac77e20f
 github.com/hashicorp/vault/api
-# github.com/hashicorp/vault/sdk v0.1.14-0.20201001203959-0ff44e5a3c48
+# github.com/hashicorp/vault/sdk v0.1.14-0.20201015192012-a69ee0f65a28
 github.com/hashicorp/vault/sdk/database/dbplugin
+github.com/hashicorp/vault/sdk/database/dbplugin/v5
+github.com/hashicorp/vault/sdk/database/dbplugin/v5/proto
 github.com/hashicorp/vault/sdk/database/helper/connutil
 github.com/hashicorp/vault/sdk/database/helper/credsutil
 github.com/hashicorp/vault/sdk/database/helper/dbutil
 github.com/hashicorp/vault/sdk/database/newdbplugin
-github.com/hashicorp/vault/sdk/database/newdbplugin/proto
 github.com/hashicorp/vault/sdk/database/newdbplugin/testing
```

Comment:
`go mod tidy && go mod vendor` gets rid of these for me.

Reply:
Done